cmd/sync.go

```diff
@@ -195,11 +195,17 @@ func supportHTTPS(name, endpoint string) bool {
 // Check if uri is local file path
 func isFilePath(uri string) bool {
 	// check drive pattern when running on Windows
-	if runtime.GOOS == "windows" &&
-		len(uri) > 1 && (('a' <= uri[0] && uri[0] <= 'z') ||
-		('A' <= uri[0] && uri[0] <= 'Z')) && uri[1] == ':' {
-		return true
+	if runtime.GOOS == "windows" {
```

Comment:
There's a more direct way. Recovery isFilePath and only use `strings.Contains(srcURL, "://")`  to infer whether it is a local path.